### PR TITLE
Feature/storage show messages

### PR DESCRIPTION
**Pull request by cdsandoval**
#7

### CODE REVIEW

File: public/assets/js/socket.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+pushingData = (text, obj, user) => {
+  obj.message.general.messages.push({
+    text,
+    date: new Date(),
+    Author: user
+  });
+};
+
+let chat = document.getElementById("chat");
+let btn = document.getElementById("btn");
```
Comment (yummta) [inline public/assets/js/socket.js]: I recomend that your classes in html and your variables like "btn" be more especific. e.g. btn-chat-send-message. This names should be more expresives and descriptive.

File: public/assets/js/socket.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+document.addEventListener("DOMContentLoaded", function() {
+  let local_storage = localStorage.getItem("data");
+  let data = JSON.parse(local_storage);
+
+  if (typeof local_storage !== "object") {
+    data.message.general.messages.map(value => {
```
Comment (yummta) [inline public/assets/js/socket.js]: Why use map if you don't return anythig?

File: public/assets/js/socket.js (hunk 1 above)
Comment (cristianbgp) [reply public/assets/js/socket.js]: Also if those are variables that have data from the DOM, we could use the symbol $ before the name of the variable

Comment (cdsandoval) [reply public/assets/js/socket.js]: This a good advise, going to do it when have to put it in the main.js.

File: public/assets/js/socket.js (hunk 2 above)
Comment (cdsandoval) [reply public/assets/js/socket.js]: I'm going to use forEach instead of map. Thanks for let me know this.

**Pull request merged**